pkg/client/services.go

```diff
@@ -84,7 +85,11 @@ func (c *services) Create(svc *api.Service) (result *api.Service, err error) {
 	if needNamespace && len(namespace) == 0 {
 		namespace = api.NamespaceDefault
 	}
-	err = c.r.Post().Namespace(namespace).Resource("services").Body(svc).Do().Into(result)
+	// TODO(a-robinson): Remove this explicit timeout and the ones below once
+	// #5180 is in.
+	request := c.r.Post()
+	request.Timeout(4 * time.Minute)
```

Comment:
That was just a nit. There were 3 places where the same timeout happened. I know it will be removed soon, but I guess it may be modified before;) Anyway @quinton-hoole is resposible for the merge.